### ISSUE TITLE
Screenshots and other images should have alt text

### ISSUE THREAD

**Issue by jensimmons** [GoogleChrome/developer.chrome.com#1]
I'm using this page as an example of content as we work: https://developers.google.com/chrome-developer-tools/docs/elements. So I've been looking at the HTML of the page. 

None of the images have alt text. They really should. 

This isn't a development bug, it's a content workflow / CMS tooling / culture issue. Something to talk about. There are a lot of people who are both professional developers and screen reader users. I personally know two. I'd love to see Google's Chrome Developer documentation include them.
